### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by laurentS**
Upgrade a bunch of libraries, this fixes no bugs, unfortunately :)

### CODE REVIEW

File: src/shared/common/BottomProgress.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // @flow
 import * as React from 'react';
-import * as Progress from 'react-native-progress';
+import ProgressBar from 'react-native-progress/Bar';
```
Comment (Hagellach37) [inline src/shared/common/BottomProgress.js:3]: Is there a good way to check for things like this? Or just trying the app and noticing that some errors will occur?

Comment (laurentS) [reply src/shared/common/BottomProgress.js:3]: check the release notes, that's the ideal way :)

**Comment by Hagellach37** [comment]
Hey @laurentS ,
maybe a general question and more in idea than actual solution: In which situations would it make sense to go back to an earlier version of a library?

For instance we could try this for react-native-redux or another library related to the bug.

File: src/shared/common/BottomProgress.js (hunk 1 above)
Comment (Hagellach37) [reply src/shared/common/BottomProgress.js:3]: okay good.

**Review by Hagellach37 — APPROVED** [review]

**Pull request merged**